apps/microtvm/zephyr/template_project/microtvm_api_server.py

```diff
@@ -455,6 +476,15 @@ def generate_project(self, model_library_format_path, standalone_crt_dir, projec
             os.makedirs(extract_path)
             tf.extractall(path=extract_path)
 
+        # Add CMSIS libraries if required.
+        if options["project_type"] == "host_driven":
```

Comment:
why only for host-driven?

Reply:
I changed it to support both and different module names.